ddtrace/tracer/startup.go

```diff
@@ -0,0 +1,114 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
The name of this file could be misleading. It's only logging, whereas now you could think it's essential stuff going on for tracer startup. Should it all go into `tracer.go`. I noticed in the Go stdlib there's not much care for file size (see `net/http` package)

Reply:
Cool, let's go with `log.go` and for the rest maybe `osinfo_[OS].go`